server/miljø.ts

```diff
@@ -41,6 +42,7 @@ const lokaltMiljø: Environment = {
   saksbehandlingstiderUrl: 'https://www.ekstern.dev.nav.no/saksbehandlingstider',
   utbetalingsoversiktUrl: 'https://www.nav.no/utbetalingsoversikt',
   utvidetBarnetrygdUrl: 'https://www.nav.no/utvidet-barnetrygd',
+  klageUrl: 'https://www.nav.no/klage',
 };
 
 const devMiljø: Environment = {
```

Comment:
Bør disse miljøene refaktoreres? Er en del duplikat kode

Reply:
Kanskje man kunne lagd en default `Environment` også overskrevet kun der det er avvik fra standard?

Reply:
Default environment bør isåfall være prod, for å unngå risikoen med "feil" urler i prod...